source/brailleDisplayDrivers/handyTech.py

```diff
@@ -24,7 +24,25 @@
 import bdDetect
 import time
 import datetime
+from ctypes import windll
+import windowUtils
 from driverHandler import BooleanDriverSetting
+import wx
+
+class InvisibleDriverWindow(windowUtils.CustomWindow):
+	className = u"Handy_Tech_Server"
+	def __init__(self, driver):
+		super(InvisibleDriverWindow, self).__init__(u"Handy Tech Server")
+		self.driver = driver
+	def windowProc(self, hwnd, msg, wParam, lParam):
+		if msg == window_message:
+			if wParam == 100 and lParam == 1:
+				self.driver.go_to_sleep()
+			elif wParam == 100 and lParam == 0:
+				self.driver.wake_up()
+		return 0
+
+window_message=windll.user32.RegisterWindowMessageW(u"Handy_Tech_Server")

```

Comment:
This is now global to the driver module and therefore to NVDA as a whole. It would be nice if we could avoid this by registering the window message in create_message_window and unregister it in self.destroy_message_window